### ISSUE TITLE
[AdMob] Accessing GADMaxAdContentRating.general makes linker failed

### ISSUE THREAD

**Issue by aplusm** [firebase/firebase-ios-sdk#2055]
* Xcode version:  9.4.1
  * Firebase SDK version: 5.11.0
  * Firebase Component: AdMob 
  * Component version: Google-Mobile-Ads-SDK (7.35.1)

Trying to set a custom max ad rating results in a failed linker process:

```
GADMobileAds.configure(withApplicationID: "MY_APP_ID")
GADMobileAds.sharedInstance().requestConfiguration.maxAdContentRating = GADMaxAdContentRating.general
```

gives:
```
Undefined symbols for architecture arm64:
  "_GADMaxAdContentRatingGeneral", referenced from:..
ld: symbol(s) not found for architecture arm64
```

**Comment by paulb777** [comment]
Tracked internally at b/119199475

**Comment by paulb777** [comment]
Fixed in Google-Mobile-Ads-SDK 7.35.2
